litmus-portal/frontend/public/locales/en/translation.yaml

```diff
@@ -191,3 +194,11 @@ workflows:
   templates: Templates
   analytics: Analytics
   scheduleAWorkflow: Schedule a workflow
+
+#################################################
+############    Air-Gapped Screens    ###########
+#################################################
+
+airGapped:
```

Comment:
give it a more generic name like `noInternet`

Reply:
updated